secretariat/tests/test_user_model.py

```diff
@@ -0,0 +1,135 @@
+from unittest import skipUnless
+
+import requests
+from django.test import TestCase
+
+from config.settings import OUTLINE_API_TOKEN, OUTLINE_API_URL, OUTLINE_OPI_GROUP_ID
+from secretariat.models import User
+from secretariat.tests.factories import UserFactory
+
+DEFAULT_OUTLINE_UUID = "not a valid uuid"
+
+
+@skipUnless(
+    OUTLINE_API_TOKEN and OUTLINE_API_URL and OUTLINE_OPI_GROUP_ID,
+    "Skip test in case of missing outline configuration",
+)
+class TestUser(TestCase):
+    headers = {
+        "Content-Type": "application/json",
+        "Accept": "application/json",
+        "Authorization": f"Bearer {OUTLINE_API_TOKEN}",
+    }
+    test_user = User(
+        first_name="prudence",
+        last_name="crandall",
+        username="pcrandall",
+        email="prudence.crandall@gouv.fr",
+        password="whatever",
+    )
+
+    def list_outline_users(self, query):
+        response = requests.post(
+            url=f"{OUTLINE_API_URL}/users.list",
+            headers=self.headers,
+            json={
+                "offset": 0,
+                "limit": 25,
+                "sort": "updatedAt",
+                "direction": "DESC",
+                "query": query,
+                "filter": "all",
+            },
+        )
+        return response
+
+    def test_creating_user_adds_them_to_outline(self):
+        user = UserFactory()
+        query = f"{user.first_name} {user.last_name}"
+        response = self.list_outline_users(query=query)
+        self.assertEqual(
+            response.json()["pagination"]["total"],
+            0,
+            f"Found an existing outline user with query {query}",
```

Comment:
Je crois que je vais garder l'habitude d'écrire des messages de test sous la forme "xxx should xxxx" pour que ce soit à la fois lisible quand on lit le code du test, et compréhensible quand le test échoue.